### PR TITLE
Update: Add metadata to existing rules - Batch 2 (refs #5417)

### PR DESCRIPTION
**Pull request by vitorbal**
Add metadata to the existing rules. Batch 2 of 11.

### CODE REVIEW

File: lib/rules/eqeqeq.js [hunk 1]
```diff
@@ -11,91 +11,101 @@
...
-     * @returns {boolean} if one of the operands is typeof
-     * @private
-     */
-    function isTypeOfBinary(node) {
-        return isTypeOf(node.left) || isTypeOf(node.right);
-    }
+module.exports = {
+    meta: {
+        docs: {
+            description: "require the use of `===` and <code>!==</code>",
```
Comment (ilyavolodin) [inline lib/rules/eqeqeq.js]: Backticks didn't convert properly here for some reason.

File: lib/rules/guard-for-in.js [hunk 2]
```diff
@@ -9,24 +9,34 @@
 // Rule Definition
 //------------------------------------------------------------------------------
 
-module.exports = function(context) {
+module.exports = {
+    meta: {
+        docs: {
+            description: "require `for-in` loops to include an <code>if</code> statement",
```
Comment (ilyavolodin) [inline lib/rules/guard-for-in.js]: Same here. Looks like second set of backticks gives your script trouble.

Comment (vitorbal) [inline lib/rules/guard-for-in.js]: yikes! you are completely right. My bad. Fixed now!

**Comment by ilyavolodin** [comment]
LGTM. Thanks!

**Pull request merged**